frontend/src/components/OperatorSidePanel.js

```diff
@@ -166,39 +136,111 @@ const OperatorSidePanel = ({ operator, showInstall, updateChannel, updateVersion
     return createdAt;
   };
 
-  return (
-    <div className="oh-operator-page__side-panel">
-      {showInstall && !allowInstall ? (
+  render() {
+    const { operator, showInstall } = this.props
+
+    const {
+      name,
+      provider,
+      capabilityLevel,
+      links,
+      channel,
+      channels,
+      version,
+      repository,
+      containerImage,
+      createdAt,
+      maintainers,
+      categories
+    } = operator;
+
+    const activeChannel = _.find(channels, { name: channel });
+    const versions = _.get(activeChannel, 'versions', [version]);
+    const currentVersion = _.find(versions, { name: _.get(activeChannel, 'currentCSV') });
+    const allowInstall = name === _.get(currentVersion, 'name');
+    const repoLink = repository ? <ExternalLink href={repository} text={repository} /> : notAvailable;
+    const tooltipText = this.state.copied ? 'Copied' : 'Copy to Clipboard';
+    const tooltipContent = [
+      <span className="oh-nowrap" key="nowrap">
+        {tooltipText}
+      </span>
+    ]
+    const tooltipOverrides = Object.freeze({
+      wrapper: {
+        cursor: 'pointer',
+        top: '2px'
+      },
+      tooltip: {
+        maxWidth: '170px',
+        minWidth: 'auto'
+      }
+    })
+
+    const imageLink = containerImage ? <Tooltip content={tooltipContent} styles={tooltipOverrides}>
+      <a
+        href="#"
+        onClick={e => this.copyToClipboard(e, containerImage)}
+        onMouseEnter={this.onCopyEnter}
+      >
+        {containerImage}
+        <Icon type="fa" name="clipboard" />
+      </a>
```

Comment:
Only the Icon should be the clickable anchor, the text should just be in a normal span.

Reply:
Ok will do